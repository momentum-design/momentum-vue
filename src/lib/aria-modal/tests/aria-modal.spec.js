import { shallowMount } from '@vue/test-utils';
import AriaModal from '../index.vue';
import FocusTrap from '../../focus-trap/index.vue';
import Vue from 'vue';

Vue.component(FocusTrap.name, FocusTrap);

describe('AriaModal', () => {
  it('should match snapshot', () => {
    const wrapper = shallowMount(AriaModal, {
      propsData: {
        titleText: 'test',
        getApplicationNode: () => global.document.querySelector('#test')
      }
    });
    expect(wrapper.html()).toMatchSnapshot();
  });
});

