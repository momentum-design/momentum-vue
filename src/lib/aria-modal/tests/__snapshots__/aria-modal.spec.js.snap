// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AriaModal should match snapshot 1`] = `
<md-focus-trap-stub active="true" focustrapoptions="[object Object]">
  <div style="position: fixed; top: 0px; left: 0px; width: 100%; height: 100%; z-index: 1050; overflow-x: hidden; overflow-y: auto; text-align: center; background: rgba(0, 0, 0, 0.5); cursor: pointer;">
    <div role="dialog" id="aria-modal-dialog" aria-label="test" style="display: inline-block; text-align: left; top: 0px; max-width: 100%; cursor: default;"></div>
  </div>
</md-focus-trap-stub>
`;
